app/services/FtneaSummaryRowBuilder.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import models.requests.DataRequest
+import models.viewmodels.checkAnswers._
+import play.api.i18n.Messages
+import play.api.mvc.AnyContent
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+
+trait FtneaSummaryRowBuilder {
```

Comment:
This is more like a Util object rather than a service. Maybe we can move the object to the util package

```suggestion
trait FtneaHelper {
```